packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -590,6 +590,7 @@ export class GitQuickOpenService {
         return this.progressService.withProgress('', 'scm', fn);
     }
 
+    protected readonly doWrapWithProgress = (fn: (...args: []) => Promise<void>) => this.wrapWithProgress(fn);
```

Comment:
@jbicker, could you turn this around to avoid changing the call sites?

```
    protected readonly wrapWithProgress = this.doWrapWithProgress.bind(this);
    protected doWrapWithProgress<In, Out>(fn: (...args: In[]) => Promise<Out>): (...args: In[]) => Promise<Out> {
        return (...args: In[]) => this.withProgress(() => fn(...args));
    }
```

Also, please do me favor and fix the `buildDefaultProjectPath` as well 🙈 

```
    private buildDefaultProjectPath = this.doBuildDefaultProjectPath.bind(this);
    private async doBuildDefaultProjectPath(folderPath: string, gitURI: string): Promise<string> {...}
```